book_1/chapter_1.md

```diff
@@ -17,27 +17,27 @@ At half past eight, the chonklord that is Mr. Dursley picked up his briefcase, p
 
 " — bet, their boi, Harry —"
 
-Mr Dursley felt like he'd been ratioed irl. He looked back at the whisperers and wondered if he should @ them, but thought better of it. Even a girlboss has his weak moments... and anyway, there were plenty of people called Potter with moots called Harry. There was no point in worrying his discord kitten- her sister was on both their DNI lists.
+Mr. Dursley felt like he'd been ratioed irl. He looked back at the whisperers and wondered if he should @ them, but thought better of it. Even a girl boss has his weak moments... and anyway, there were plenty of people called Potter with moots called Harry. There was no point in worrying his discord kitten- her sister was on both their DNI lists.
 
-During that day, emo thoughts kept dabbing across Mr Dursley's mind. He later left the building and nearly sister slammed into someone just outside his door.
+During that day, emo thoughts kept dabbing across Mr. Dursley's mind. He later left the building and nearly sister slammed into someone just outside his door.
 
-"Naurrr," he grunted, as the man nearly fell. However, he didn't seem triggered at all- instead, the funky little dude pressed the slay button. "So true, bestie! Even Muggles like u should slayy today, when The Sussiest Of Bakas has been canceled at last!"
+"Naurrr," he grunted, as the man nearly fell. However, he didn't seem triggered at all; instead, the funky little dude pressed the slay button. "So true, bestie! Even Muggles like u should slayy today, when the Sussiest Of Bakas has been canceled at last!"
 
-And then the old man hugged Mr Dursley (real) and headed off.
+And then the old man hugged Mr. Dursley (real) and headed off.
```

Comment:
Apparently the British use `Mr Dursley` without the full stop.

Harry Potter and the Philosopher's Stone has it as `Mr Dursley`.
Harry Potter and the Sorcerer's Stone has it as `Mr. Dursley`.

I think it might be best to leave it as is given the origin of the source material, however a case can be made for keeping consistency with the previous text.